crates/gosub_pipeline/viewpage.sh

```
@@ -0,0 +1,6 @@
+#/usr/bin/bash
+
+. tools/souper/bin/activate
+python tools/souper/soupertoo.py $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)